### PR TITLE
repo-add: add page

### PR DESCRIPTION
**Pull request by Shinyzenith**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/linux/repo-add.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# repo-add
+
+> Package database maintenance utility which enables installation via pacman.
```
Comment (sbrl) [inline pages/linux/repo-add.md:3]: Does this mean that:

1. It enables me to install a given package file with `pacman`
2. pacman can't work without it

File: pages/linux/repo-add.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+- Add all package binaries in the current directory and remove the old database file:
```
Comment (sbrl) [inline pages/linux/repo-add.md:5]: Add all package binaries in the current directory to where?

**Review by sbrl — CHANGES_REQUESTED** [review]
Thanks for the new page! I've left some comments below for you to review.

File: pages/linux/repo-add.md (hunk 2 above)
Comment (Shinyzenith) [reply pages/linux/repo-add.md:5]: To a database file in the same directory as shown in the following image. To put it up 1 directory, you'd run ../yourdatabasename.db.tar.gz
image

File: pages/linux/repo-add.md (hunk 1 above)
Comment (Shinyzenith) [reply pages/linux/repo-add.md:3]: Number 1. I'll cleanup that part.

Comment (Shinyzenith) [reply pages/linux/repo-add.md]: https://github.com/tldr-pages/tldr/pull/6622/commits/3aadcab4afac46d4fed555d76dfcb65fa95db507 I hope the following commit helps.

File: pages/linux/repo-add.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# repo-add
+
+> Package database maintenance utility which enables installation of said package via pacman.
+
+- Add all package binaries in the current directory and remove the old database file:
+
+`repo-add --remove {{your_database_name.db.tar.gz *.pkg.tar.zst}}`
```
Comment (marchersimon) [inline pages/linux/repo-add.md:7]:
```suggestion
`repo-add --remove {{path/to/database.db.tar.gz}} {{*.pkg.tar.zst}}`
```

File: pages/linux/repo-add.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+- Add all package binaries in the current directory in silent mode except for warning and error messages:
+
+`repo-add --quiet {{your_database_name.db.tar.gz *.pkg.tar.zst}}`
```
Comment (marchersimon) [inline pages/linux/repo-add.md:11]:
```suggestion
`repo-add --quiet {{path/to/database.db.tar.gz}} {{*.pkg.tar.zst}}`
```

File: pages/linux/repo-add.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+- Remove color from output:
+
+`repo-add --nocolor {{your_database_name.db.tar.gz *.pkg.tar.zst}}`
```
Comment (marchersimon) [inline pages/linux/repo-add.md:15]:
```suggestion
- Add all package binaries in the current directory without showing color:

`repo-add --nocolor {{path/to/database.db.tar.gz}} {{*.pkg.tar.zst}}`
```

File: pages/linux/repo-add.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+# repo-add
+
+> Package database maintenance utility which enables installation of said package via pacman.
```
Comment (navarroaxel) [inline pages/linux/repo-add.md:3]:
```suggestion
> Package database maintenance utility which enables installation of said package via Pacman.
> More information: <https://man.archlinux.org/man/repo-add>.
```

**Review by marchersimon — APPROVED** [review]
Thanks @Shinyzenith

**Review by sbrl — APPROVED** [review]
Still don't really understand what this does, but thanks anyway, @Shinyzenith! :D

**Pull request merged**